### ISSUE TITLE
AttributeError with wxPython 4.0.3

### ISSUE THREAD

**Issue by tradel** [wxFormBuilder/wxFormBuilder#451]
wxFormBuilder: 3.8.0
wxPython: 4.0.3

I created a frame and added a DataViewListCtrl and a few columns to it. wxFormBuilder generates the following code:
```
self.m_dataViewListCtrl2 = wx.dataview.DataViewListCtrl( self, wx.ID_ANY, wx.DefaultPosition, wx.DefaultSize, wx.dataview.DV_SINGLE )
self.account_col = self.m_dataViewListCtrl2.AppendTextColumn( u"Account", wx.DATAVIEW_CELL_INERT, -1, wx.ALIGN_LEFT, wx.DATAVIEW_COL_RESIZABLE ) 
self.opp_col = self.m_dataViewListCtrl2.AppendTextColumn( u"Opportunity", wx.DATAVIEW_CELL_INERT, -1, wx.ALIGN_LEFT, wx.DATAVIEW_COL_RESIZABLE ) 
self.amount_col = self.m_dataViewListCtrl2.AppendTextColumn( u"Amount", wx.DATAVIEW_CELL_INERT, -1, wx.ALIGN_LEFT, wx.DATAVIEW_COL_RESIZABLE ) 
self.close_date_col = self.m_dataViewListCtrl2.AppendTextColumn( u"Close Date", wx.DATAVIEW_CELL_INERT, -1, wx.ALIGN_LEFT, wx.DATAVIEW_COL_RESIZABLE ) 
bSizer2.Add( self.m_dataViewListCtrl2, 1, wx.ALL|wx.EXPAND, 5 )
```

The code will not run with the latest version of wxPython. It results in an error:
```
Traceback (most recent call last):
  File "/Users/tradel/Code/main.py", line 48, in OnInit
    self.main_frame = MainFrame(None)
  File "/Users/tradel/Code/gui.py", line 35, in __init__
    self.account_col = self.m_dataViewListCtrl2.AppendTextColumn( u"Account", wx.dataview.wx.DATAVIEW_CELL_INERT, -1, wx.ALIGN_LEFT, wx.DATAVIEW_COL_RESIZABLE ) 
AttributeError: module 'wx' has no attribute 'DATAVIEW_CELL_INERT'
OnInit returned false, exiting...
```

It appears that the `DATAVIEW_CELL_*` constants have moved to the `wx.dataview` package in 4.0.3.

**Comment by MauricioFarias** [comment]
I have updated from 3.8.0 to 3.8.1 and it has begun to happen. 
Before update only the first parameter `AppendTextColumn( u"Label", wx.DATAVIEW_CELL_INERT, -1, wx.ALIGN_LEFT, wx.DATAVIEW_COL_RESIZABLE ) ` was generated. After update the last parameters appear.

**Comment by redcat22** [comment]
I have the same problem, how can I solve the issue please?

**Comment by ivanpu** [comment]
My workaround currently includes a sed script, that I run after each generation of the code.
So far I have only this rule:
```sed
s/wx\.DATAVIEW/wx.dataview.DATAVIEW/g
```
and more can be added, if needed.

**Comment by sebreit** [comment]
Had the same problem. Since there is currently no solution for Issue #459: "Hardcoded modules in Python and Lua code generators", this could be solved by patching pythoncg.cpp in the meantime:

```
--- c:/data/Projekte/wxFormBuilder/wxFormBuilder-master/src/codegen/pythoncg.cpp.bak	Mon Aug 26 12:21:32 2019
+++ c:/data/Projekte/wxFormBuilder/wxFormBuilder-master/src/codegen/pythoncg.cpp	Tue Aug 27 11:56:13 2019
@@ -1847,6 +1847,18 @@
 	ADD_PREDEFINED_PREFIX( wxPGMAN_DEFAULT_STYLE, wx.propgrid. );
 	ADD_PREDEFINED_PREFIX( wxPG_NO_INTERNAL_BORDER, wx.propgrid. );
 
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_ACTIVATABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_EDITABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_FOCUSED, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_INERT, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_INSENSITIVE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_PRELIT, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_SELECTED, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_HIDDEN, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_REORDERABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_RESIZABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_SORTABLE, wx.dataview. );
+
 	ADD_PREDEFINED_PREFIX( wxDV_SINGLE, wx.dataview. );
 	ADD_PREDEFINED_PREFIX( wxDV_MULTIPLE, wx.dataview. );
 	ADD_PREDEFINED_PREFIX( wxDV_ROW_LINES, wx.dataview. );

```

**Comment by KyleHunter** [comment]
Is there an official fix for this yet? Been an issue for a couple years now.

**Comment by sodevel** [comment]
Should be fixed by 630972f6b51df4a9b60fb1c265a3912beefc6a8c.
